### PR TITLE
Create quick start for databricks pipeline

### PR DESCRIPTION
**Pull request by janet-can**

### CODE REVIEW

File: soda/quick-start-databricks-pipeline.md [hunk 1]
```diff
@@ -0,0 +1,518 @@
...
+       dimension: [Consistency]
+       pipeline_stage: Ingest
+       team: Data Engineering
+
+
+checks for employee_survey:
+ - invalid_count(EnvironmentSatisfaction) = 0:
+     valid min: 1
+     valid max: 5
+     name: Values are formatted are in range 1-5
```
Comment (vivi-belogianni) [inline soda/quick-start-databricks-pipeline.md:171]: Values are formatted in range 1-5 (my bad)

File: soda/quick-start-databricks-pipeline.md [hunk 2]
```diff
@@ -0,0 +1,518 @@
...
+ - duplicate_count(EmployeeID) = 0:
+     name: No duplicate IDs
+     attributes:
+       dimension: [Uniqueness]
+       pipeline_stage: Ingest
+       team: Data Engineering
+ - invalid_count(WorkLifeBalance) = 0:
+     valid min: 1
+     valid max: 5
+     name: Values are formatted are in range 1-5
```
Comment (vivi-belogianni) [inline soda/quick-start-databricks-pipeline.md:191]: same here

File: soda/quick-start-databricks-pipeline.md [hunk 3]
```diff
@@ -0,0 +1,518 @@
...
+
+# Import Scan from Soda Library
+from soda.scan import Scan 
+import yaml
+from io import StringIO
+from pathlib import Path
+from datetime import datetime, timedelta
+
+# Define file directory
+settings_path = Path('/Workspace/Users/vivi.belogianni@soda.io/employee_attrition/soda_settings')
```
Comment (vivi-belogianni) [inline soda/quick-start-databricks-pipeline.md:358]: would it make sense here to replace my actual path with a dummy one?

File: soda/quick-start-databricks-pipeline.md [hunk 4]
```diff
@@ -0,0 +1,518 @@
...
+
+# Define results file directory
+result_path = Path('/Workspace/Users/vivi.belogianni@soda.io/employee_attrition/checks_output')
```
Comment (vivi-belogianni) [inline soda/quick-start-databricks-pipeline.md:361]: same here

File: soda/quick-start-databricks-pipeline.md [hunk 5]
```diff
@@ -0,0 +1,518 @@
...
+partition = (datetime.today().date() - timedelta(days=1)).strftime("%Y-%m-%d")
+# Create a scan object
+scan = Scan()
+
+# Set scan name and data source name
+scan.set_scan_definition_name("Employee Attrition Scan")
+scan.set_data_source_name("employee_info")
+
+# Add file to be scanned 
+df = spark.read.option("header", True).csv(f"dbfs:/Workspace/Users/vivi.belogianni@soda.io/employee_attrition/soda_settings/login_logout/PartitionDate={partition}")
```
Comment (vivi-belogianni) [inline soda/quick-start-databricks-pipeline.md:373]: same here

File: soda/quick-start-databricks-pipeline.md [hunk 6]
```diff
@@ -0,0 +1,518 @@
...
+    for check in metadata.get('checks', [])
+]
+
+# output_data = {
+#     'scanEndTimestamp':metadata['scanEndTimestamp'],
+#     'checks': checks_data
+# }
+import json
+checks_data_json = json.dumps(checks_data)
+scan.save_scan_result_to_file(result_path/f"test.json", checks_data_json)
```
Comment (vivi-belogianni) [inline soda/quick-start-databricks-pipeline.md:430]: this part can be deleted as it was just for testing. Probably made some changes before you finished working on it :)

File: soda/quick-start-databricks-pipeline.md [hunk 7]
```diff
@@ -0,0 +1,518 @@
...
+%restart_python
+
+# Import Scan from Soda Library
+from soda.scan import Scan 
+import yaml
+from io import StringIO
+from pathlib import Path
+
+# Define file directory
+settings_path = Path('/Workspace/Users/vivi.belogianni@soda.io/employee_attrition/soda_settings')
```
Comment (vivi-belogianni) [inline soda/quick-start-databricks-pipeline.md:452]: same here for the path

File: soda/quick-start-databricks-pipeline.md [hunk 8]
```diff
@@ -0,0 +1,518 @@
...
+
+#### Data Ingestion Checks
+Download: <a href="/assets/Data Ingestion Checks.ipynb" download>Data Ingestion Checks.ipynb</a>
+{% include code-header.html %}
+```python
+# Install to run checks on data in the Unity catalog
+pip install -i https://pypi.cloud.soda.io soda-spark-df
+
+# Install to run checks on data in the file stored in Databricks file system
+pip install -i https://pypi.cloud.soda.io soda-spark[databricks]
```
Comment (vivi-belogianni) [inline soda/quick-start-databricks-pipeline.md:347]: I believe here the soda-spark-df is to run checks on top of files and the soda-spark[databricks] to run checks on top of tables in unity

File: soda/quick-start-databricks-pipeline.md [hunk 9]
```diff
@@ -0,0 +1,518 @@
...
+```
+
+<br />
+
+#### Input Data Checks
+Download: <a href="/assets/Input Data Checks.ipynb" download>Input Data Checks.ipynb</a>
+{% include code-header.html %}
+```python
+# Install to run checks on data in the file stored in Databricks file system
+pip install -i https://pypi.cloud.soda.io soda-spark[databricks]
```
Comment (vivi-belogianni) [inline soda/quick-start-databricks-pipeline.md:440]: I think here the comment should be to run checks on tables in unity

File: soda/quick-start-databricks-pipeline.md (hunk 3 above)
Comment (janet-can) [reply soda/quick-start-databricks-pipeline.md:358]: Yes, agreed. I missed that! This is why I never publish the same day I prepare something :)

File: soda/quick-start-databricks-pipeline.md (hunk 4 above)
Comment (janet-can) [reply soda/quick-start-databricks-pipeline.md:361]: Got it. I did a find+replace.

File: soda/quick-start-databricks-pipeline.md (hunk 8 above)
Comment (janet-can) [reply soda/quick-start-databricks-pipeline.md:347]: Ah, I got it mixed up. I've corrected.

**Pull request merged**